src/transformers/models/auto/feature_extraction_auto.py

```diff
@@ -77,6 +77,7 @@
         ("perceiver", "PerceiverFeatureExtractor"),
         ("poolformer", "PoolFormerFeatureExtractor"),
         ("pop2piano", "Pop2PianoFeatureExtractor"),
+        ("propainter", "ProPainterFeatureExtractor"),
```

Comment:
Can be omitted, feature extractors are deprecated

Reply:
done